services/masterdata/src/lib/core/services/masterdata-helper.service.ts

```diff
@@ -0,0 +1,17 @@
+import { Injectable } from "@angular/core";
+import { TranslateService } from "@ngx-translate/core";
+
+@Injectable()
+export class MasterdataHelperService {
+	constructor(
+		private readonly translateService: TranslateService
+	) {}
+
+	prepareForDisplay<T extends object>(item: T, translationKey: string) {
+		// "metadata" model field will have a special display!
+		return Object.entries(item).filter(item => item[0] !== "metadata").map(item => {
+			item[0] = this.translateService.instant(`masterdatamgmt.pages.${translationKey}.modelFields.${item[0]}`);
+			return item;
```

Comment:
Why is it needed to return item at line 14? You're also returning Object.entries on line 12. Seems unreachable code to me

Reply:
@Johanh-od good point, you are right!